wagtail_transfer/static_src/components/ContentImportForm/index.js

```diff
@@ -127,6 +132,7 @@ export default function ImportContentForm({
         sourcePage={sourcePage}
         onClick={onClickSubmit}
         disabled={!destPage}
+        numPages = {numPages}
```

Comment:
spacing around the `=` inconsistent here

Reply:
Could you run `npm run format` and commit the results? Should fix this automatically

Reply:
I've run and seems to have fixed, thanks.